src/server.ts

```diff
@@ -30,6 +31,10 @@ const { routes, routePaths } = getRoutes()
 app.get(
   routePaths,
   async (req: ArtsyRequest, res: ArtsyResponse, next: NextFunction) => {
+    if (ENABLE_SSR_STREAMING) {
+      res.write("<html>")
```

Comment:
sneaky!

Reply:
ran into some error-page edge-cases with this :/ need to experiment more.